playground/ios/playground.xcodeproj/project.pbxproj

```diff
@@ -1091,7 +1091,7 @@
 				DEBUG_INFORMATION_FORMAT = dwarf;
 				GCC_C_LANGUAGE_STANDARD = gnu11;
 				INFOPLIST_FILE = SnapshotTests/Info.plist;
-				IPHONEOS_DEPLOYMENT_TARGET = 13.2;
+				IPHONEOS_DEPLOYMENT_TARGET = 13.0;
```

Comment:
We're downgrading to 13.0?

Reply:
It's only for the snapshotTests target, matched it to the deployment target used in CI